### ISSUE TITLE
Does not work with Uplay

### ISSUE THREAD

**Issue by Mikestriken** [MAXBURAOT/SmartSteamEmu#6]
I know this isn't supposed to work with uplay, but I just wanted to know if it would be at all possible for a uplay variant of this to be available?

**Comment by segg21** [comment]
@Mikestriken The Crew I assume?

**Comment by Mikestriken** [comment]
Nope, can't remember what, but definitely not the crew (not really a race-format game fan)

**Comment by segg21** [comment]
Lol. Yea, SSE is useless for you, and only for steam related games. I had The Crew 2, but can't crack it because of steam + uplay lol. Oh well
